data/incipias/incipias ships.txt

```diff
@@ -0,0 +1,239 @@
+		"cargo space" 20
+		"outfit space" 552
+		"weapon capacity" 240
+		"engine capacity" 130
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Liquid-Class Shield"
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 2
+		"Star Tail" 4
+		"Star Tail Cell" 80
+		"Star Tail Storage" 2
+		"Cloud Piercer" 40
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+	engine -90.5 42
+	engine 0 42
+		"over"
+	engine 90 42
+	gun 58.5 -76.5
+	gun 29 -76.5
+	gun 29 -76.5
+	gun -28 -76.5
+	gun -28 -76.5
+	gun -57 -76
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "When the Hicemus came to power they needed a ship to symbolize their strength. With six guns, the Venta is the most powerful ship in the Incipias' history. Thanks to its sleek and aerodynamic design it is as comfortable hunting down outlaws in space as in the clouds of a gas giant."
+
+ship "Nimbus"
+	"plural" "Nimbi"
+	"sprite" ship/nimbus
+	"thumbnail" thumbnail/nimbus
+	"swizzle" 0
+	attributes
+		"category" "Medium Warship"
+		licenses
+			"Hicemus Conflict"
+		"cost" 500000
+		"hull" 5000
+		"shields" 7500
+		"required crew" 20
+		"bunks" 40
+		"mass" 1640
+		"drag" 10.8
+		"heat dissipation" .46
+		"cargo space" 75
+		"outfit space" 450
+		"weapon capacity" 120
+		"engine capacity" 85
+		"gaslining" 1
+		"fuel capacity" 200
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam"
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+	engine 0 120
+	engine 0 120
+	engine -30.5 120
+	engine 30.5 120
+	gun -15 -117
+	gun 15 -117
+	gun 0 -117
+	gun 0 -117
+	turret 0 31.5
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Nimbus is the first spaceship that the Incipias built that was truly able to navigate through a system. Since it was first manufactured it has been cherished among wealthier captains and a dream of captains just starting out."
+	description "Although the Nimbus is labeled as a medium warship, it is multipurpose; whether one uses it for mining, passenger transport or cargo, it will perform its job nicely. It is only occasionally used in combat because the Incipias are not currently fighting open wars."
+
+ship "Nimbus" "Nimbus (Civil)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbus" "Nimbus (Mining)"
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Plasma Discharger" 4
+		"Stagnation Beam" # to be replaced with a flotsam tractor beam
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+ship "Nimbo Stratus"
+	"plural" "Strati"
+	"sprite" ship/stratus
+	"thumbnail" thumbnail/stratus
+	"swizzle" 0
+	attributes
+		"category" "Utility"
+		"cost" 50000
```

Comment:
This costs less than a Scrapper (60000), despite being much more capable in every area. I'd suggest upping this to roughly 500,000ish, broadly in line with larger interceptors.

Reply:
```suggestion
		"cost" 460000
```